kart/cli.py

```diff
@@ -233,6 +246,210 @@ def push(ctx, do_progress, args):
     )
 
 
+@cli.command(context_settings=dict(ignore_unknown_options=True))
```

Comment:
Consider moving all the helper stuff to a separate file. cli.py is getting to be a mix of things - the implementations of any commands that are too small to have their own file, plus the kart-helper, plus some other miscellany, but these things are unrelated to each other.

Reply:
Sure thing.